dvc/repo/add.py

```diff
@@ -77,7 +100,12 @@ def _find_all_targets(repo, target, recursive):
 def _create_stages(repo, targets, fname, pbar=None):
     stages = []
 
-    for out in targets:
+    for out in Tqdm(
+        targets,
+        desc="Creating stages",
```

Comment:
We try not to call them "stages" in public 😄 Same as our docs, we should call them "DVC-files" instead

Reply:
done